test/onigumo_cli_test.exs

```diff
@@ -35,6 +42,20 @@ defmodule OnigumoCLITest do
       assert Onigumo.CLI.main(["downloader"]) == tmp_dir
     end
 
+    @tag :tmp_dir
+    test("run CLI 'downloader' with '--working-dir' switch", %{tmp_dir: tmp_dir}) do
+      expect(OnigumoDownloaderMock, :main, fn root_path -> root_path end)
+
+      assert Onigumo.CLI.main(["downloader", "--working-dir", tmp_dir]) == tmp_dir
+    end
+
+    @tag :tmp_dir
+    test("run CLI 'downloader' with '-C' switch", %{tmp_dir: tmp_dir}) do
+      expect(OnigumoDownloaderMock, :main, fn root_path -> root_path end)
+
+      assert Onigumo.CLI.main(["downloader", "-C", tmp_dir]) == tmp_dir
+    end
+
```

Comment:
I am missing a test verifying what happens if you pass in the `--working-dir`/`-C` switch without a value. Like

```
$ ./onigumo downloader --working-dir
```

Does it use the default, or does it pass an empty string? Is it a valid use case?